draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1363,7 +1411,7 @@ The following procedure implements the simplified SWU algorithm in a
 straight-line fashion. This implementation is optimized for the case
 that q = 3 (mod 4), which applies to P-256.
 For discussion of how to generalize to q = 1 (mod 4), see
-{{WB19}} (Section 4) or the example code found at {{github-repo}}.
+{{WB19}} (Section 4) or the example code found at {{hash2curve-repo}}.
```

Comment:
This implementation is optimized for the case that q = 3 (mod 4), which applies to P-256
This implementation is optimized for the case that q = 3 (mod 4). ~~which applies to P-256~~

Reply:
Can we leave this alone until we go back through and do curve-specific reference code, then update all of these examples? The reason is that there are similar examples given in other mappings, so it would be better to make this change systematically.

Reply:
Added a note in #111 so that we don't lose this.